TopDownProteomics.nuspec

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>TopDownSDK</id>
+    <version>0.0.1</version>
+    <title>TopDownSDK</title>
+    <authors>Center for Top-Down Proteomics</authors>
+    <owners>Center for Top-Down Proteomics</owners>
```

Comment:
Should be "Consortium for Top-Down Proteomics"

Reply:
Oops, thanks! Good catch.